contracts/contracts/src/functioncall/interface/CrosschainFunctionCallInterface.sol

```diff
@@ -33,4 +33,6 @@ interface CrosschainFunctionCallInterface {
         address _contract,
         bytes calldata _functionCallData
     ) external;
+
+    function isSingleBlockchainCall() external view returns (bool);
```

Comment:
This API is for applications calling to the function call layer, indicating that they want to make a crosschain call.

Reply:
As such, the isSingleBlockchainCall doesn't belong in this contract interface

Reply:
An additional reason for not having isSingleBlockchainCall in this API is that this API is for both GPACT and SFC. With SFC (and other non-atomic protocols), function calls are executed the same whether they are single blockchain calls or not. As such, I don't there is any advantage of SFC Crosschain Control contract implementations providing this function.